examples/example9/continuous.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import scipy.sparse as sps
+
+from porepy.viz import plot_grid, exporter
+from porepy.fracs import meshing
+
+from porepy.params import tensor
+from porepy.params.bc import BoundaryCondition
+from porepy.params.data import Parameters
+
+from porepy.grids import coarsening as co
+
+from porepy.numerics.vem import dual
+
+#------------------------------------------------------------------------------#
+
+def define_data(g):
+    domain = {'xmin': np.amin(g.nodes[0, :]),
+              'xmax': np.amax(g.nodes[0, :]),
+              'ymin': np.amin(g.nodes[1, :]),
+              'ymax': np.amax(g.nodes[1, :])}
+    tol = 1e-5
+
+    param = Parameters(g)
+
+    # Permeability
+    kxx = np.ones(g.num_cells)
+    perm = tensor.SecondOrder(g.dim, kxx)
+    param.set_tensor("flow", perm)
+
+    # Boundaries
+    b_faces = g.get_boundary_faces()
+    b_face_centers = g.face_centers[:, b_faces]
+
+    top = b_face_centers[1, :] > domain['ymax'] - tol
+    bot = b_face_centers[1, :] < domain['ymin'] + tol
+    left = b_face_centers[0, :] < domain['xmin'] + tol
+    right = b_face_centers[0, :] > domain['xmax'] - tol
+
+    outer = np.logical_or.reduce((top, bot, left, right))
+    inner = np.logical_not(outer)
+
+    labels = np.array(['dir'] * b_faces.size)
+
+    bc_val = np.zeros(g.num_faces)
+    bc_val[b_faces[inner]] = 1
+
+    param.set_bc("flow", BoundaryCondition(g, b_faces, labels))
+    param.set_bc_val("flow", bc_val)
+
+    return {'param': param}
+
+#------------------------------------------------------------------------------#
+
+def glue_tips(partition, gb, g):
+    seeds = co.generate_seeds(gb)
+
+    left = seeds[g.cell_centers[0, seeds] < 0.5]
+    right = seeds[g.cell_centers[0, seeds] > 0.5]
+
+    partition[partition == partition[left[1]]] = partition[left[0]]
+    partition[partition == partition[right[1]]] = partition[right[0]]
+
+#------------------------------------------------------------------------------#
+
+folder = '/home/elle/ANIGMA/porepy/examples/exampleXX/'
```

Comment:
you're right. I just add this example to be on the safe side, it's not really in it's final version but almost. The same for your next comment.